dns/src/http.rs

```diff
@@ -0,0 +1,53 @@
+mod models;
+mod routes;
+
+use crate::config::Config;
+use actix_web::{web, web::Data, App, HttpRequest, HttpServer};
+use anyhow::{anyhow, Error};
+use rate_limit::backend::{memory::InMemoryBackend, SimpleInputFunctionBuilder};
+use std::time::Duration;
+
+pub use models::Domain;
+
+pub fn get_token<'a>(req: &'a HttpRequest) -> Result<(&'a str, &'a str), Error> {
+    let header = match req.headers().get("authorization") {
```

Comment:
Ensure you are catching both `authorization` and `Authorization`

Reply:
Yeah, this one for sure. Headers are finicky with case sensitivity, but [Mozilla standardized case-insensitivity](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers)

Reply:
Actix auto-handles the header spelling